netgrasp/database/database.py

```diff
@@ -1,72 +1,74 @@
-from netgrasp.utils import debug
+import datetime
+
 from netgrasp.utils import exclusive_lock
 
-class Database:
-    def __init__(self):
-        from netgrasp import netgrasp
-        ng = netgrasp.netgrasp_instance
 
-        try:
-            import sqlite3
-        except Exception as e:
-            ng.debugger.error("Error: %e")
-            ng.debugger.critical("Failed to import sqlite3, try: 'pip install sqlite3', exiting.")
-        
-        self.connection = sqlite3.connect(ng.database["filename"], detect_types=sqlite3.PARSE_DECLTYPES)
+class Database(object):
 
-    def set_state(self, key, value, secret = False):
-        from netgrasp import netgrasp
-        ng = netgrasp.netgrasp_instance
```

Comment:
What value is there in maintaining a local copy of all of these values? They're already available through netgrasp_instance. These changes all look unrelated to adding local MAC support.